### ISSUE TITLE
Links: Lack global focus styles

### ISSUE THREAD

**Issue by tskarhed** [grafana/grafana#36474]
There is a lack of global link styles, which is an accessibility issue.
___
(Old part of the issue)

**Steps to reproduce**:
1. Try to tab through the home dashboard

**Actual Result**:

![Screenshot from 2021-07-06 14-28-57](https://user-images.githubusercontent.com/1438972/124600264-d2d2e880-de66-11eb-8724-ad169174d1e1.png)

**Expected Result**

To have focus styles present around the link elements. This could possible be a wider Grafana issue than just this panel.

**Comment by tharun208** [comment]
@tskarhed, I like to work on this

**Comment by tskarhed** [comment]
Awesome @tharun208! There is a function called `getFocusStyles`, which might be useful
https://github.com/grafana/grafana/blob/b5cdf5fdd8c13bb76cbba1e6cc31af70f2e2c69f/packages/grafana-ui/src/themes/mixins.ts#L56

**Comment by tharun208** [comment]
@tskarhed, any mockup for the focus style's or color from the `palette` to use based on the theme would be helpful

**Comment by tskarhed** [comment]
The focus styles look like this. 
![Screenshot from 2021-07-07 09-25-28](https://user-images.githubusercontent.com/1438972/124717765-bafe7180-df05-11eb-886a-2975f60c6576.png)
If you want to see how other components have implemented the focus styles, the RadioButtonGroup is a good example.
https://github.com/grafana/grafana/blob/7538739881673c0ef0a4eeac4a9ae21bdf56db0a/packages/grafana-ui/src/components/Forms/RadioButtonGroup/RadioButton.tsx#L77
